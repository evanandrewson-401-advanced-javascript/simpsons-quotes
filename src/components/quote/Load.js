import React from 'react';
import PropTypes from 'prop-types';

const Load = ({ fetchQuote }) => {
  return <button onClick={fetchQuote} >Get New Quote</button>;
};

Load.propTypes = {
  fetchQuote: PropTypes.func.isRequired
};

export default Load;
