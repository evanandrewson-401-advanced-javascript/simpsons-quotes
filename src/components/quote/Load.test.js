import React from 'react';
import { shallow } from 'enzyme';
import Load from './Load';

describe('Load component', () => {
  it('renders Load component', () => {
    const wrapper = shallow(<Load fetchQuote={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
