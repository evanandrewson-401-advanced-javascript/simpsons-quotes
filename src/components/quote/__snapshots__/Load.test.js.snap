// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Load component renders Load component 1`] = `
<button
  onClick={[Function]}
>
  Get New Quote
</button>
`;
